source/server/listener_manager_impl.cc

```diff
@@ -220,6 +220,12 @@ ListenerImpl::ListenerImpl(const envoy::api::v2::Listener& config, const std::st
 
   for (const auto& filter_chain : config.filter_chains()) {
     const auto& filter_chain_match = filter_chain.filter_chain_match();
+    if (!filter_chain_match.address_suffix().empty() || filter_chain_match.has_suffix_len() ||
+        filter_chain_match.source_prefix_ranges_size() || filter_chain_match.source_ports_size()) {
```

Comment:
Should there be tests for each of these cases? Not sure if it's worth the boilerplate, but it's a bit weird to have explicit rejections that we don't test.

Reply:
I think it's not worth the extra tests, since those are already unsupported edge cases, and we have the line coverage from one of them, but I'm happy to add them if you prefer, let me know.